scenes/World/World.tscn

```diff
@@ -18,7 +18,7 @@ layer = -20
 [node name="Planet" parent="." groups=["Planets"] instance=ExtResource( 3 )]
 
 [node name="Pilot" parent="." instance=ExtResource( 2 )]
-position = Vector2( 960, 540 )
+position = Vector2( 605, 473 )
```

Comment:
There are two positions to consider in regards to our player:

1. The Pilots position within the world scene
2. The ships position within the pilot scene

I adjusted the starting values so that the pilot starts offset a bit, causing the planet to show up on camera to the right side of the screen. I also defaulted the ship to be at position 0, 0. This made the coordinates easier to track.

Reply:
Agree that ship position should have always been 0,0. I personally liked that when you had the option to choose a pilot it used to appear over the planet because the pilot's position was set to the same coordinates as the planet's to cause this effect and then updated when a ship was instanced in (you deleted that line of code). In the Pilot scene the position should be set to 0,0 - for aesthetic reasons I vote for the pilot instance in world scene to be set to the planet's and updated in code.

Reply:
It's worth mentioning that this interaction happens because the pilot scene carries the camera. There are other methods to achieve the same effect.

Reply:
I put this back to the way it was. Let me know if it looks like how you want it. I think the reason I change it was because there's a camera jump when you pick your player. Maybe in the future we can make that a smooth camera transition rather than a jump.